### PR TITLE
[#177] fix 404 in readme

### PR DESCRIPTION
**Pull request by pfeifferj**
updating url based on docs from: https://github.com/openshift/aws-ebs-csi-driver/blob/master/docs/install.md#installation-1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ To build and run the operator locally:
 
 ```shell
 # Create only the resources the operator needs to run via CLI
-oc apply -f https://raw.githubusercontent.com/openshift/cluster-storage-operator/master/assets/csidriveroperators/aws-ebs/10_cr.yaml
+oc apply -k "github.com/kubernetes-sigs/aws-ebs-csi-driver/deploy/kubernetes/overlays/stable/?ref=release-1.11"
```
Comment (jsafrane) [inline README.md:26]: This URL will install upstream CSI driver, not the operator. I think https://github.com/openshift/cluster-storage-operator/blame/master/assets/csidriveroperators/aws-ebs/standalone/10_cr.yaml is the correct location of the CR for the operator now.

Comment (pfeifferj) [reply README.md]: good catch, I corrected the url

**Comment by jsafrane** [comment]
/label docs-approved
/label px-approved
/label qe-approved
This is just readme change

**Pull request merged**